models/spleen_ct_segmentation/configs/inference.json

```diff
@@ -153,7 +153,8 @@
         "load_path": "$@bundle_root + '/models/model.pt'",
         "load_dict": {
             "model": "@network"
-        }
+        },
+        "map_location": "@device"
```

Comment:
change here to fix cpu device issue when create bundle workflow. Other bundles that use "checkpointloader" may also have the same issues. I will update them when updating new monai version. cc @KumoLiu